### PR TITLE
Add plugin via Crac feature

### PR DESCRIPTION
**Pull request by timyates**
Will not work until the plugin is released

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -39,20 +33,6 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Checkout the utils
```
Comment (sdelamo) [inline .github/workflows/gradle.yml]: If I understand correctly with this change we only test our docker build tasks?  Would not be better to test both?

**Review by sdelamo — COMMENTED** [review]
should

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (timyates) [reply .github/workflows/gradle.yml]: You're right, I added it back in

I also added a summary so we can see how long things took to run

https://github.com/micronaut-projects/micronaut-crac-tests/actions/runs/3174048446

image

Comment (sdelamo) [reply .github/workflows/gradle.yml]: This looks great thanks.

**Pull request merged**